src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -218,7 +218,7 @@ public void forceGC() {
 	 * @return the unique identifier of the currently running instance
 	 */
 	@Bean
-	public UUID promregatorInstanceIdentifer() {
+	public UUID promregatorInstanceIdentifier() {
```

Comment:
What a luck that `@Autowired` also works on type basis.
(nothing to be changed here!)